gatsby-config.js

```diff
@@ -128,7 +128,7 @@ module.exports = {
         background_color: `#fff`,
         theme_color: `#33b2e8`,
         display: `minimal-ui`,
-        icon: `content/images/scm-manager_logo_img.png`, // This path is relative to the root of the site.
+        icon: `content/images/scm-manager_logo_favicon.png`, // This path is relative to the root of the site.
```

Comment:
According to the docs one should provide a base-image with 512px. All used sized will be derived by the module. A non-square image is fine as it will be filled with additional transparent parts until square.

Reply:
Thanks, haven't thought of this.